test/integration/02-user-wallet/02-invoice.spec.ts

```diff
@@ -32,7 +32,7 @@ describe("UserWallet - addInvoice", () => {
   })
 
   it("adds a self generated invoice without amount", async () => {
-    const lnInvoice = await addInvoiceNoAmountForSelf({
+    const lnInvoice = await addInvoiceNoAmount({
```

Comment:
the error was not detected because this was addInvoice, so while we move to the new graphql all previous related tests must be done at graphql level. please add them to `test/integration/servers/graphql-main-server.spec.ts` or add an `addInvoice` wrapper method with the logic to accept null/undefinied amount

Reply:
I suspected this would be a problem, check out the one open task included in this PR's description :sweat_smile:

Reply:
Should be fixed here: https://github.com/GaloyMoney/galoy/pull/463/commits/1ab2d83b0a320c03031f0b8f1bb3d0bd522a508e